src/main/java/duke/FindCommand.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+/**
+ * Represents the command to find tasks containing a particular keyword.
+ * 
+ * @author Sherman Ng Wei Sheng
+ */
+public class FindCommand extends Command {
+    
+    private boolean isExit;
+    private String keyword;
+
+    /**
+     * Constructor for the command object.
+     * 
+     * @param keyword The keyword to be searched for.
+     */
+    public FindCommand(String keyword) {
+        this.isExit = false;
+        this.keyword = keyword;
+    }
+    
+    /**
+     * Returns true if the command is a programme terminating command.
+     *
+     * @return True if it is a terminating command and false otherwise.
+     */
+    @Override
+    public boolean isExit() {
+        return false;
```

Comment:
Maybe replace false with isExit (even though they are always the same)
On a side note, do you think the boolean isExit is a little bit redundant? (Since only ExitCommand will lead to the termination of the program)